docs/python-reference/dtype.ipynb

```diff
@@ -6,8 +6,7 @@
    "source": [
     "# Data types\n",
     "\n",
-    "Scipp supports the most common data types (`dtype`).\n",
-    "In most cases the `dtype` is derived from the data, e.g., when passing a [numpy.ndarray](https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.html#numpy.ndarray) to scipp, scipp will use `dtype` provided by numpy:"
+    "In most cases, the data type (`dtype`) of a [Variable](../generated/scipp.Variable.html) is derived from the data. For instance when passing a [numpy array](https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.html#numpy.ndarray) to scipp, scipp will use the `dtype` provided by numpy:"
```

Comment:
We are generally trying to put one sentencce per line in docs/markdown to improve diff readability.